src/coord/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
 cmake_minimum_required(VERSION 3.13)
 
 get_filename_component(root ${CMAKE_CURRENT_LIST_DIR}/../.. ABSOLUTE)
-include(${root}/deps/readies/cmake/main)
-
+# Removed readies dependency
```

Comment:
we can remove this comment. we don't want any readies in our code base, including comments ;)